api/nodes.go

```diff
@@ -394,7 +394,7 @@ func (n *Nodes) Allocations(nodeID string, q *QueryOptions) ([]*Allocation, *Que
 
 func (n *Nodes) CSIVolumes(nodeID string, q *QueryOptions) ([]*CSIVolumeListStub, error) {
 	var resp []*CSIVolumeListStub
-	path := fmt.Sprintf("/v1/csi/volumes?node_id=%s", nodeID)
+	path := fmt.Sprintf("/v1/volumes/csi?type=csi&node_id=%s", nodeID)
```

Comment:
Based on the rest of this PR, shouldn't this be `"/v1/volumes?type=csi&node_id=%s"`? i.e. leaving out the `csi` in the path and having it only in the query param.

Reply:
yes, thanks! 647dde18b